### DISCUSSION TITLE
Next steps for the stable-diffusion.cpp

### DISCUSSION THREAD

**Discussion by snakelizzard** [leejet/stable-diffusion.cpp#309] [General]
Hello @leejet.

Thank you for all you have done to create stable-diffusion.cpp. Can you please share your plans for the future of the project? I see Stable Diffusion 3 pull request (https://github.com/leejet/stable-diffusion.cpp/pull/298) left with no attention for some time. What are your plans for other features like mask support, faster inference etc.?

Thank you.

**Comment by leejet** [comment]
Support for sd3 has been added. My next work plan is to add support for lora and taesd to sd3, followed by adding flash attention support.
